transport.go

```diff
@@ -7,92 +7,17 @@ import (
 	"fmt"
 	"io"
 	"log"
-	"net/http"
 	"os"
 	"sync"
-	"sync/atomic"
-	"time"
 
-	"github.com/gorilla/websocket"
```

Comment:
please remove this dependency from the go.mod file?

Reply:
It is used in testing ws